packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -53,22 +76,38 @@ internal class DatadogSDKWrapper : DatadogWrapper {
     }
 
     override fun telemetryDebug(message: String) {
-        Datadog._internal._telemetry.debug(message)
+        // Do not initialize the telemetry proxy before SDK is initialized
```

Comment:
proxy won't be recreated on every telemetry event, because `lazy` does initialization only once and then memoized value is returned each time.

However, maybe we can optimize `isInitialized` call? `Datadog.isInitialized` is thread-safe call, but if this safety overhead is not needed and this wrapper is called only from the single thread always, maybe we can store initialization flag directly here?